src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1857,7 +1834,7 @@ BoundExpression createPlaceholderIfNecessary(BoundExpression expr, TypeSymbolWit
         public override BoundNode VisitConditionalReceiver(BoundConditionalReceiver node)
         {
             var result = base.VisitConditionalReceiver(node);
-            SetResult(node);
+            _resultType = TypeSymbolWithAnnotations.Create(node.Type, isNullableIfReferenceType: false, treatUnconstrainedTypeParameterAsNullable: false);
```

Comment:
>_resultType [](start = 12, length = 11)

This is a bit strange. The conditional receiver itself doesn't have a non-null result type.
Feels like this logic should be moved up into the conditional access logic.

Reply:
Added `// PROTOTYPE` comment.

---
In reply to: [208683066](https://github.com/dotnet/roslyn/pull/29127#discussion_r208683066) [](ancestors = 208683066)